sjb/config/test_cases/test_pull_request_ose_new.yml

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
I was expecting just to see a shell script -- we can't generate and push out to that jenkins master